eraser-map.yaml

```diff
@@ -794,6 +794,16 @@ styles:
         base: points
         texture: pois
         interactive: true
+        
+    icons-overlay:
+        base: points
+        texture: pois
+        interactive: true
+        blend: overlay
```

Comment:
It looks like the `icons-overlay` style is still in there in `gh-pages`; it's harmless, but would be good to clean out at some point, along with the `interactive` parameters mentioned above. 

I'm not totally clear what you mean by moving `lines-overlay` "above" `icons`, but I'll mention that the ordering of styles in the scene file doesn't guarantee drawing order.